src/includes/configuration/before-send-fingerprint/java.spring-boot.mdx

```diff
@@ -1,12 +1,13 @@
 ```java
 import io.sentry.SentryEvent;
 import io.sentry.SentryOptions;
+import io.sentry.hints.Hint
```

Comment:
Should we move `Hint` to `io.sentry` package so devs don't have to deal with different packages? Or is it OK like this?

Reply:
PR open https://github.com/getsentry/sentry-java/pull/2064